### PR TITLE
Don't create broadcasts for contact replies

### PR DESCRIPTION
**Pull request by rowanseymour**
`ReplyAction` / `Contact.send` / `Msg.reply` shouldn't create broadcasts

Also:
* Namespace priority constants inside classes
* Remove redundant update of `modified_on` from `Msg` bulk action methods

### CODE REVIEW

**Comment by nicpottier** [comment]
Yarp

**Comment by rowanseymour** [comment]
Had to make lots of changes to _channels/tests.py_ as all our handler tests make heavy use of `Contact.send(...)` - so many that the diff doesn't even appear here. Got a bit confused by the way those tests typically pass a stale version of the message to `Channel.send_message` to make it act like a first attempt, and ended up changing things to always refresh from the db. But looking at it now, I still think that's preferable as it ensures fields like `error_count` are actually being updated.

File: temba/msgs/models.py [hunk 1]
```diff
@@ -570,6 +566,10 @@ class Msg(models.Model):
 
     MEDIA_TYPES = [MEDIA_AUDIO, MEDIA_GPS, MEDIA_IMAGE, MEDIA_VIDEO]
 
+    PRIORITY_HIGH = 1000
+    PRIORITY_NORMAL = 500
+    PRIORITY_BULK = 100
```
Comment (ericnewcomer) [inline temba/msgs/models.py]: wooo!

**Comment by ericnewcomer** [comment]
Yerp!

**Pull request merged**